src/Model/RequestManager.php

```diff
@@ -39,15 +40,39 @@ public function selectAllRequests(): ?array
               town.name AS townName, 
               town.postal_code AS townPostalCode, request.publication_date AS requestPublicationDate 
               FROM ' . self::TABLE .
-            ' JOIN ' . UserManager::TABLE . ' ON user.id = fk_requester_id ' .
-            ' JOIN ' . AddressManager::TABLE . ' ON address.id = fk_town_id ' .
-            ' JOIN ' . TownManager::TABLE . ' ON town.id = fk_address_id ' .
-            ' JOIN ' . MeasurementManager::TABLE . ' ON measurement.id = fk_measurement_id')->fetchAll();
+            ' LEFT JOIN ' . UserManager::TABLE . ' ON user.id = fk_requester_id ' .
+            ' LEFT JOIN ' . AddressManager::TABLE . ' ON address.id = fk_address_id ' .
+            ' LEFT JOIN ' . TownManager::TABLE . ' ON town.id = fk_town_id ' .
+            ' LEFT JOIN ' . MeasurementManager::TABLE . ' ON measurement.id = fk_measurement_id')->fetchAll();
         } catch (\PDOException $error) {
             return null;
         }
         if ($statement !== false) {
             return $statement;
         }
     }
+
+    public function insert(Request $request) : void
+    {
+        if ($request->getQuantity() === '') {
+            $request-> setQuantity(null);
+        }
+        if ($request->getMeasurementId() === '') {
+            $request-> setMeasurementId(null);
+        }
+        if ($request->getDescription() === '') {
+            $request-> setDescription(null);
+        }
```

Comment:
If you need to have null instead of "" for there's no quantity or no description, this should probably be done in the Request class